src/backend/fs.rs

```diff
@@ -225,7 +196,7 @@ pub async fn read_file(pk: &Secp256k1PubKey, blake3_hash: &Blake3Hash) -> Result
                     let mut bytes = vec![];
                     file.read_to_end(&mut bytes).unwrap();
 
-                    let (_header, chunk) = bytes.split_at(Header::len() as usize);
+                    let (_header, chunk) = bytes.split_at(Header::len());
```

Comment:
Why usize is no longer necessary?

Reply:
Because the return type for len was changed upstream in carbonado 0.3.0-rc.7